modules/core/api/module.go

```diff
@@ -16,27 +16,37 @@ type IBCModule interface {
 	// for this specific application.
 	OnSendPacket(
 		ctx context.Context,
-		sourceID string,
-		destinationID string,
+		sourceChannel string,
+		destinationChannel string,
 		sequence uint64,
 		data channeltypesv2.PacketData,
 		signer sdk.AccAddress,
 	) error
 
 	OnRecvPacket(
 		ctx context.Context,
-		sourceID string,
-		destinationID string,
+		sourceChannel string,
+		destinationChannel string,
 		data channeltypesv2.PacketData,
 		relayer sdk.AccAddress,
 	) channeltypesv2.RecvPacketResult
 
 	// OnTimeoutPacket is executed when a packet has timed out on the receiving chain.
 	OnTimeoutPacket(
 		ctx context.Context,
-		sourceID string,
-		destinationID string,
+		sourceChannel string,
```

Comment:
Something funky in the diff here, I didn't add these many lines, the OnAcknowledgementPacket function was already added here: https://github.com/cosmos/ibc-go/pull/7438/files#diff-c1878b54cff15b20121c65b05f9710b3d9082166ef34b3815e2bf00677040fbfR42-R51